src/components/AppBreadcrumbs/AppBreadcrumbs.tsx

```diff
@@ -7,7 +7,7 @@ type Context = { type: BreadcrumbType; id: string; name: string };
 type Breadcrumb = { text: string; to: string };
 
 type AppBreadcrumbsProps = {
-  childBreadcrumbs: Breadcrumb[];
+  childBreadcrumbs?: Breadcrumb[];
```

Comment:
Are we really expecting this to ever be null?

Reply:
Yeah, I think probably forcing an empty list (if that's our default anyways) makes sense